README.md

```diff
@@ -91,6 +91,7 @@ Advanced Options:
 	-p, --pin=<wps pin>             Use the specified pin (may be arbitrary string or 4/8 digit WPS pin)
 	-d, --delay=<seconds>           Set the delay between pin attempts [1]
 	-l, --lock-delay=<seconds>      Set the time to wait if the AP locks WPS pin attempts [60]
+	-I, --ignore-reason             Ignore checking of reason code for NACK [False]
```

Comment:
i'd rather just always do the right thing than adding yet another command line option.